test/book/specialized/fang-fourier-monomial-bases/oscar-vs-gap.jlcon

```diff
@@ -0,0 +1 @@
+basis_lie_highest_weight(:A, 4, [1,3,2,1]);
```

Comment:
There is a "bug" here in the bug (just fixed it there):
```suggestion
julia> basis_lie_highest_weight(:A, 4, [1,3,2,1]);
```

Reply:
@benlorenz @lkastner so should this file (and the other two "non-code" jlcon files) be put into a "do not include list" somewhere and dropped here?